mlir/test/Dialect/Linalg/convert-elementwise-to-linalg.mlir

```diff
@@ -19,6 +19,53 @@ func.func @addf_rank0(%arg0: tensor<f32>, %arg1: tensor<f32>) -> tensor<f32> {
   return %0 : tensor<f32>
 }
 
+// Test a binary elementwise op with a tensor and a scalar operand.
+// CHECK-LABEL: func @addf_tensor_plus_scalar_rank1
+//  CHECK-SAME:   %[[T:[0-9a-zA-Z]*]]: tensor<?xf32>, %[[S:[0-9a-zA-Z]*]]: f32
+func.func @addf_tensor_plus_scalar_rank1(%t: tensor<?xf32>, %s: f32) -> tensor<?xf32> {
```

Comment:
These tests still don't exercise the new logic. The rewrite occurs on `arith` ops that have tensor only operands.
I'm not sure if there are any upstream ops which are both `ElementwiseMappable` and accept mixed scalar/shape type operands.

Reply:
After a bit of digging, I've found there's this test op that should also work with this conversion pass:
```mlir
func.func @test(%arg0: f32, %arg1: tensor<?x?xf32>) {
  %0 = "test.elementwise_mappable"(%arg0, %arg1) : (f32, tensor<?x?xf32>) -> tensor<?x?xf32>
  return
}
```